lib/rajska.ex

```diff
@@ -85,58 +84,50 @@ defmodule Rajska do
       def get_user_role(%{role: role}), do: role
       def get_user_role(nil), do: nil
 
-      def user_role_names, do: unquote(roles_names)
-
       def default_rule, do: unquote(default_rule)
 
-      def valid_roles, do: [:all | user_role_names()]
-
-      def not_scoped_roles, do: [:all | unquote(super_roles)]
+      def valid_roles, do: [:all | unquote(valid_roles)]
 
-      defguard super_role?(role) when role in unquote(super_roles)
```

Comment:
why was this guard removed?

Reply:
It was replaced by:

```elixir
def super_role?(unquote(super_role)), do: true

def role_authorized?(unquote(super_role), _allowed_role), do: true

Reply:
I recreated the `is_super_role` guard, so it can be used in the user created Authorization module